drivers/jason0x43-wemo_switch.groovy

```diff
@@ -179,3 +188,84 @@ private debugLog(message) {
 private syncTime() {
     parent.childSyncTime(device)
 }
+
+
+def updatePort(port) {
```

Comment:
The code for this function should ideally go in the parent app so that all the drivers can use it. This function would then just call `parent.childUpdatePort(child)`.